config/software/libyaml-windows.rb

```diff
@@ -25,13 +25,13 @@
 # of rubyinstaller.org
 #
 name "libyaml-windows"
-default_version "0.1.7"
+default_version "0.2.2"
 
-version "0.1.7" do
-  source sha256: "c90f4a678a7901bf0c48fc50cd6ffbd96eeb0b0184ba6c853fc0f54702213674"
+version "0.2.2" do
+  source sha256: "9d430d3788081027a2dcf13fb8823b5ee296b1c8fe0353c86339b4c7b4018441"
 end
 
-source :url => "https://s3.amazonaws.com/dd-agent/libyaml-#{version}-x64-windows.tar.lzma",
+source :url => "https://s3.amazonaws.com/dd-agent-omnibus/libyaml-#{version}-x64-windows.zip",
```

Comment:
Where was this downloaded / how was this built ?

Reply:
We pre-built it and uploaded there. It would be better to build it from omnibus, but this is what we were already doing until now.